### ISSUE TITLE
No release file for the Ubuntu Lunar (23.04) Release

### ISSUE THREAD

**Issue by VMmmTV** [TheAssassin/AppImageLauncher#601]
### Pre-submit checks

- [X] I [checked for similar issues](https://github.com/TheAssassin/AppImageLauncher/issues?q=) beforehand, but could not find any, not even closed ones. I could not add my bug report to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Describe the bug

When adding the PPA to Lubuntu 23.04 XFCE by executing the command: 
- sudo add-apt-repository ppa:appimagelauncher-team/stable 
the repository won't update due to the error:  
- No release file for the ubuntu lunar Release
Therefore the install doesn't proceed.

### Expected behavior

Expected the repository to update so the install can be done.

### Steps to reproduce the issue

Add the PPA executing the following commands:

sudo apt install software-properties-common
sudo add-apt-repository ppa:appimagelauncher-team/stable
sudo apt update
sudo apt install appimagelauncher

### Screenshots

Error shown in Terminal:

![](http://imgur.com/LHcXN7Z.png)

### Distribution and desktop environment

Distro: Lubuntu 23.04 
DE: XFCE

### Installed AppImageLauncher version

AppImage didn't install so don't know the version.

### Additional context

I'm creating an instructional video about using AppImageLauncher in Lubuntu 23.04

**Comment by vmvartiainen** [comment]
Having this same problem with Kubuntu 23.10

**Comment by KlfJoat** [comment]
BTW, this is a dupe of #587 & #571 

And nothing so far for Ubuntu Minotaur (23.10) release.

Between this and the other obvious bugs (adding Exec line options that prevent the AppImage from executing and putting icons in `0x0` directories), I'm just going to abandon it.

**Comment by MMachado05** [comment]
> Between this and the other obvious bugs (adding Exec line options that prevent the AppImage from executing and putting icons in 0x0 directories), I'm just going to abandon it.

Yeah, it seems like there are a lot of recent unfixed bugs in this repo and not a lot of recent involvement from TheAssasin. I mean, the last commit was in July, which is about 3 months ago now. Is this thing dead? AppImages are already kind of a pain to deal with, it would suck if the (currently) only way to actually get them to play nice on our machine was just up and abandoned.

**Comment by TheAssassin** [comment]
Nope, not dead, preparing the next release, which is going to fix a whole bunch of problems with the deployment. The PPA is not my department, I just occasionally fix bugs. I don't have nearly as much time to spend on this. The entire AppImage project is maintained by less than a handful of people, and this project is just maintained by me.

@RogueScholar could you please have a look?

**Comment by TheAssassin** [comment]
The build recipe seems broken for the newer distros. You can always install the `.deb` from GitHub, though.

**Comment by cstmth** [comment]
Would be great if that was reflected in the docs.I would gladly open a PR but I do not really understand the necessary context of why this workaround is necessary.

On the bright side, directly installing the current .deb from releases page and installing via apt worked like a charm. :)

## Workaround

For those stumbling upon this issue, follow the instructions of [the wiki page on installing directly from GitHub](https://github.com/TheAssassin/AppImageLauncher/wiki/Install-on-Ubuntu-or-Debian#installing-directly-from-github)

Short and dirty way:

1. Download .deb file from latest release
2. Install with `sudo apt install ./your-downloaded-deb-file.deb`
3. Profit!

**Comment by anvishinc** [comment]
> Would be great if that was reflected in the docs.I would gladly open a PR but I do not really understand the necessary context of why this workaround is necessary.
> 
> On the bright side, directly installing the current .deb from releases page and installing via apt worked like a charm. :)
> 
> ## Workaround
> For those stumbling upon this issue, follow the instructions of [the wiki page on installing directly from GitHub](https://github.com/TheAssassin/AppImageLauncher/wiki/Install-on-Ubuntu-or-Debian#installing-directly-from-github)
> 
> Short and dirty way:
> 
> 1. Download .deb file from latest release
> 2. Install with `sudo apt install ./your-downloaded-deb-file.deb`
> 3. Profit!

That is one method, you could also use the offcial PPA and after adding it just change the release from lunar to jammy, and also download the deb file for libappimage0 and install that before installing the launcher. More work but if the first option fails try this.

**Comment by TheAssassin** [comment]
There's the continuous prerelease that works reasonably stable. I'm working on providing new binaries for a new release, but that requires major changes to how the software is built. It cannot happen overnight. I'm doing my best.

**Comment by stuaxo** [comment]
Thanks for all the work, this stuff takes time :)

**Comment by Petemir** [comment]
> There's the continuous prerelease that works reasonably stable. I'm working on providing new binaries for a new release, but that requires major changes to how the software is built. It cannot happen overnight. I'm doing my best.

Yes, I understand it and very much appreciate the work on maintaining this! The launcher's capabilities have been of great help. I just wanted to express my confusion about the `.deb` versions; I didn't know the prelease was considered stable, in that case it should be good enough  then (at least for me).

**Comment by carsonbird** [comment]
> There's the continuous prerelease that works reasonably stable. I'm working on providing new binaries for a new release, but that requires major changes to how the software is built. It cannot happen overnight. I'm doing my best.

Happy to help, if it will help. Is there a way to distribute the work across the community?
